connectors/s3-tests/src/test/resources/s3test/api_test.bal

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/config;
+import ballerina/http;
+import ballerina/kubernetes;
+import ballerina/log;
+import wso2/amazons3;
+
+// Constants for error code and messages.
+
+const string RESPOND_ERROR_MSG = "Error in responding to client.";
+const string CLIENT_CREATION_ERROR_MSG = "Error while creating the AmazonS3 client.";
+const string BUCKET_CREATION_ERROR_MSG = "Error while creating bucket on Amazon S3.";
+const string BUCKET_DELETION_ERROR_MSG = "Error while deleting bucket from Amazon S3.";
+
+// Read accessKey and secretKey from config files.
+string accessKeyId = config:getAsString("ACCESS_KEY_ID");
+string secretAccessKey = config:getAsString("SECRET_ACCESS_KEY");
+
+// Create Amazons3 client configration with the above accesskey and secretKey values.
+amazons3:ClientConfiguration amazonS3Config = {
+    accessKeyId: accessKeyId,
+    secretAccessKey: secretAccessKey,
+    clientConfig: {
+        http1Settings: {chunking: http:CHUNKING_NEVER}
+    }
+};
+
+//Add `@kubernetes:Service` to a listner endpoint to expose the endpoint as Kubernetes Service.
+@kubernetes:Service {
+    //Service type is `NodePort`.
+    serviceType: "NodePort"
+}
+//Add `@kubernetes:Ingress` to a listner endpoint to expose the endpoint as Kubernetes Ingress.
+@kubernetes:Ingress {
+    //Hostname of the service is `abc.com`.
+    hostname: "abc.com"
```

Comment:
We should have a proper hostname like `bi.wso2.com` for ingress.

Reply:
done